INSTALL.md

```diff
@@ -27,7 +27,7 @@ cd picard-website
 cp website/config.py.example website/config.py
 ```
 
-Edit website/config.py so that PLUGINS_REPOSITORY points to your local copy of the picard-plugins repository:
+Edit website/config.py so that PLUGINS_BUILD_DIR points to your local temporary directory:
```

Comment:
It is not supposed to point to a local temporary directory. It's where the plugin files will be generated to.